### ISSUE TITLE
IoT fleet deployment and management

### ISSUE THREAD

**Issue by pedrosola** [pedrosola/greengo#5]
Previous experience working with greengrass projects reveals the need to **create**, **update** and **deploy** multiple IoT replicas at once. It get's exponentially hard to do so manually. For every new device a couple of new resources are created including, but not restricted to:

- IoT Certificates
- Lambda functions
- IoT Groups
- IoT MQTT Topics

Not to mention the intricate problems when one wants to update their code or add new features to the fleet, which needs to be done individually on every IoT Group. Now, this feature should provide a big improvement on fleet management and replicating capabilities. As the developer can specify the number of clones of devices he or she wants to create. Also, when there's a new update `greengo deploy` command can **update the IoT fleet** with the new code making the development experience that much faster.

**TL;DR:** Final product should be something like:

`greengo create -n 999 (...)` - Creates 999 IoT Groups all using the same code. And let's developers launch updates simultaneously to those 999 different devices.
